src/lib/commandBarConfigs/settingsCommandConfig.ts

```diff
@@ -129,10 +125,18 @@ export function createSettingsCommand({
     icon: 'settings',
     needsReview: false,
     onSubmit: (data) => {
-      if (data !== undefined && data !== null) {
-        send({ type: `set.${type}`, data })
+      if (
+        data !== undefined &&
+        data !== null &&
```

Comment:
These are probably redundant, no? If the two below conditions are true, surely, these cannot be?

Reply:
Yeah, but if you leave the two conditions above out, `tsc` will error saying that `data` may be `undefined | null` as well, just a type guard.